ui/admin/app/controllers/scopes/scope/credential-stores/index.js

```diff
@@ -126,4 +126,39 @@ export default class ScopesScopeCredentialStoresIndexController extends Controll
     this[paramKey] = [...selectedItems];
     this.page = 1;
   }
+
+  /**
+   * Save worker filter
+   * @param {CredentialStoreModel} credentialStore
+   */
+
+  @action
+  @loading
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess('notifications.add-success')
+  async saveWorkerFilter(credentialStore) {
+    await credentialStore.save();
+    await this.router.replaceWith(
```

Comment:
I think we only need these 2 router methods together when we are deleting a resource. Since we are saving, we can just use the `replaceWith` if I am not mistaken.

Reply:
There are certain times were `refresh` is needed like in `saveWorkerFilters` for targets because aliases are only populated in a target get request but not a update request & we display aliases in the details view. In some other save actions refresh is needed so that after a resource is created, it appears in list view (ex: storage policies). However, in this case refresh is not needed for save & cancel since we are updating a field in an existing resource.